<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="My_Resources.qrc">
    <normaloff>:/icons/cloud.ico</normaloff>:/icons/cloud.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item row="1" column="1">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="downloadButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Download File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Delete File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Upload File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="undoButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Undo Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reloadButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Reload Storage</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="adminPanelButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Admin Panel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mainLayout" stretch="2,0,3,3,1">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QTreeView" name="fileViewer">
            <property name="styleSheet">
             <string notr="true">font-size: 20px;
background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="imagePreview">
            <property name="styleSheet">
             <string notr="true">width:200px;
height:100px;
background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="responseDisplay">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="updateTextFile">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="loginPage">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0" columnminimumwidth="0,0,0,0">
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="0">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>285</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="13" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="3">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>285</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="2">
         <widget class="QFrame" name="loginFrame">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(61, 43, 68);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0">
           <item row="2" column="0">
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="portInput">
             <property name="styleSheet">
              <string notr="true">background-color:rgb(61, 56, 70)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="stateLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(36, 31, 49);
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);
color:rgb(165, 29, 45);</string>
             </property>
             <property name="text">
              <string>Disconnected</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="validateButton">
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Validate Port</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="usernameInput">
             <property name="styleSheet">
              <string notr="true">background-color:rgb(61, 56, 70)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="passwordInput">
             <property name="styleSheet">
              <string notr="true">background-color:rgb(61, 56, 70)</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="submitLogin">
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Log in</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="adminPage">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QGridLayout" name="adminLayout" rowstretch="0,0" columnstretch="0,1,1" columnminimumwidth="0,0,0">
          <item row="0" column="2">
           <widget class="QTableWidget" name="userCredentialsTable">
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Username</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Password</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Admin Privileges</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2" alignment="Qt::AlignTop">
           <widget class="QFrame" name="createUserFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>476</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>476</width>
              <height>455</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(61, 43, 68);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="createAccountLayout" rowstretch="0,0,0,0,0,0,0,1" columnstretch="0" rowminimumheight="0,0,0,0,0,0,0,0">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item row="3" column="0">
                <widget class="QLineEdit" name="createAccountUsername">
                 <property name="styleSheet">
                  <string notr="true">background-color:rgb(61, 56, 70)</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLineEdit" name="createAccountPassword">
                 <property name="styleSheet">
                  <string notr="true">background-color:rgb(61, 56, 70)</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QComboBox" name="adminPrivilegesBox">
                 <property name="styleSheet">
                  <string notr="true">text-align:center;</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>True</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>False</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="Line" name="adminPrivilegesSeparator">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QPushButton" name="createAccountButton">
                 <property name="cursor">
                  <cursorShape>OpenHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Create</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
                <widget class="QLabel" name="adminLabel">
                 <property name="minimumSize">
                  <size>
                   <width>112</width>
                   <height>18</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>112</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style:none;</string>
                 </property>
                 <property name="text">
                  <string>Admin Privileges</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="mainPanelButton">
            <property name="text">
             <string>Main Panel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="My_Resources.qrc"/>
 </resources>
 <connections/>
</ui>
